pages/osx/rename.md

```diff
@@ -0,0 +1,11 @@
+# rename
+
+> Rename a file or group of files with a regex.
+
+- Replace from.txt -> to.txt:
+
+`rename 's/from/to/' *.txt`
+
+- Replace all filename spaces with underscores for all files in the current directory:
+
+`rename 's/ /_/g' *`
```

Comment:
This example seems the same as the previous one, just with a different find / replace combo.